libs/backend/src/cast_vote_records/test_utils.ts

```diff
@@ -0,0 +1,91 @@
+/* istanbul ignore file */
+import fs from 'fs';
+import path from 'path';
+import { computeCastVoteRecordRootHashFromScratch } from '@votingworks/auth';
+import { assertDefined } from '@votingworks/basics';
+import { CVR, safeParseJson } from '@votingworks/types';
+
+import { readCastVoteRecordExportMetadata } from './import';
+
+function identifyFunction<T>(input: T): T {
+  return input;
+}
+
+/**
+ * Modifies a cast vote record export. Specifically meant for modifying fixtures for tests.
+ */
+export async function modifyCastVoteRecordExport(
```

Comment:
As you'll see in a subsequent commit, this is a replacement for: https://github.com/votingworks/vxsuite/blob/3ba81d4d418df406aa5304fac2dbbc6e9cf159ae/apps/admin/backend/test/utils.ts#L13-L17